assets/content/cookbook/Intermediate/06.CustomFreeplayStyles.md

```diff
@@ -0,0 +1,45 @@
+[tags]: / "intermediate,ui,json"
+
+This chapter will walk you through the process of creating a functioning, fully compatible Friday Night Funkin' mod, using the game's official systems for loading custom content and scripts. Once your mod is complete, you will be able to place it in the `mods` folder in your game install and use its content in-game without overriding the base game content and still maintain compatibility with other mods.
```

Comment:
`This chapter will walk you through the process of creating a functioning, fully compatible Friday Night Funkin' mod...` seems a little out of place tbh, maybe you included this by mistake? Considering this is also the same excerpt from [Introduction](https://github.com/FunkinCrew/code-cookbook/blob/master/assets/content/cookbook/Introduction/01.Introduction.md)